### PR TITLE
chore: change iv to nonce in all files

### PR DESCRIPTION
**Pull request by PuBHARGAVI**
changed iv to nonce in all files except in encrypt and decrypt functions

### CODE REVIEW

**Review by vharsh — APPROVED** [review]
lgtm

@shreeThaanuTw please merge this PR after your singleton fixes are merged into develop so that merge conflict issues are reduced.

**Comment by shreethaanu** [comment]
Good to go

**Review by shreeThaanuTw — APPROVED** [review]

**Pull request merged**
